lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
Ok, that reset flag is gone now, actually I could add the `if (use_asm) throw new Error('Toplevel started with "use asm"');` error. But meh, this would be pushed to user code as well.